dessia_common/workflow/blocks.py

```diff
@@ -112,6 +113,18 @@ def dict_to_object(cls, dict_: JsonSerializable, force_generic: bool = False,
     def evaluate(_):
         return []
 
+    def _to_script(self) -> ToScriptElement:
+        input_variables = [f"Variable(memorize={input.memorize}, name='{input.name}')" for input in self.inputs]
+        inputs = '[' + ",".join(input_variables) + ']'
```

Comment:
Adding a space after the coma would match PEP 8 convention for the resulting script